specifyweb/specify/calculated_fields.py

```diff
@@ -24,7 +24,7 @@ def calculate_actual_count(obj, preparations):
         exchangeoutprep_quantity = calculate_quantity(prep.exchangeoutpreps, "quantity")
         disposalpreparation_quantity = calculate_quantity(prep.disposalpreparations, "quantity")
 
-        countamt = obj.countamt or 0
+        countamt = getattr(obj, 'countamt', 0) or 0
```

Comment:
or 0 is redundant?

Reply:
Or does it needs to be there to catch null?

Reply:
it's a better way to write
```python
countamt = obj.countamt if hasattr(obj, 'countamt') and obj.countamt is not None else 0
# is eqivalent to 
countamt = getattr(obj, 'countamt', 0) or 0
```